### DISCUSSION TITLE
resource missing from ui

### DISCUSSION THREAD

**Discussion by itsai220** [concourse/concourse#6445] [Help & Support]
Hi guys, 

I've just upgraded my concourse from 5.8.x to 6.4.1 and i'm running into couple issues that i'm not sure if i missed from the release notes or if it's an issue somewhere i need to fix.

my cluster is deployed via helm chart on a k8s cluster

1. pipeline view is gone, i've tried redeploying this and it's not coming back. I can still see resources when i use fly command to list them, however it' be nice to see the whole pipeline mapping.

2. i'm using the default postgresql db provided from the helm chart and now i can see postgresql logs 
`FATL remaining connection slots are reserved for non-replication superuser connections`
i see only about 37 idle connections, and it's not close to default 100 connections.

 Please let me know what info is required to help debug this. 

Thanks!

**Comment by itsai220** [comment]
1. is fixed, seems to be browser issue on chrome, relaunched browser and everything came back.

**Reply by aoldershaw** [reply]
Possibly the javascript was cached? A hard refresh would probably fix it in the future

**Comment by taylorsilva** [comment]
> i'm using the default postgresql db provided from the helm chart

I would recommend not using this. Use a managed postgres instance if possible, it's safer. The postgres chart is there to make it easy to get started with Concourse on k8s.

How many web nodes do you have? Not sure why postgres would be throwing that error if you already confirmed it's no where near the max number of connections. Might be a red herring?

**Reply by itsai220** [reply]
got 2 web nodes atm, with around ~70-80 connections

**Reply by itsai220** [reply]
yes this will be moved out to a managed postgres, but for now this is what we got and it was working perfectly before the upgrade .. might be red herring, i'm not sure but i wonder if others might be hitting this

**Reply by taylorsilva** [reply]
Are you noticing any weird behaviour because of those errors? Sounds like you got a pretty healthy concourse...
